packages/components/src/ui/utils/get-valid-children.ts

```diff
@@ -12,11 +12,13 @@ import { Children, isValidElement } from '@wordpress/element';
 /**
  * Gets a collection of available children elements from a React component's children prop.
  *
- * @param {import('react').ReactNode} children
+ * @param children
  *
- * @return {import('react').ReactNodeArray} An array of available children.
+ * @return An array of available children.
```

Comment:
Just to confirm, it is a good practice to keep JSDocs in TypeScript files to add a description to the function. Correct?

Reply:
Correct, we leave the descriptions (it's actually called "TSDoc" at that point) but remove the type annotations.

Reply:
Got it, thank you!